### PR TITLE
landing zone v2 documentation update

### LINKED ISSUE #383: Documentation Updates to support package renaming/structure [BUILD]
**Description**
To update documentation to support the publication of new packages in #370 

**Acceptance Criteria**

- pubsec documentation updated
- gcp-documentation updated
- Review of the document

### PR DESCRIPTION
**Pull request by davelanglois-ssc**
updating doc to support new consolidated packages
closes #383

### CODE REVIEW

File: CONTRIBUTING.md
Comment (davelanglois-ssc) [inline CONTRIBUTING.md:1]: fix linting

**Review by cartyc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM
- formally deploying the onboarding changes over late today

File: docs/landing-zone-v2/onboarding-client.md [hunk 1]
```diff
@@ -1,36 +1,97 @@
...
+    ├── core-landing-zone
+    ├── gatekeeper-policies
+    ```
+
+## <a name='Addclient-setuppackage'></a>Add client-setup package
+
+1. Get the client-setup package
+
+> **!!! Update the command below with the proper VERSION, you can locate it in the package's CHANGELOG.md, for example, '0.0.1'. Use 'main' if not available**
+
```
Comment (Michael-McAuley-SSC) [inline docs/landing-zone-v2/onboarding-client.md:44]: I am wondering about the advice to use 'main'. During version updates it was necessary to use a specific version for kpt pkg update to work on renamed target directories @stephanemillaire-ssc - lets discuss.

**Review by Michael-McAuley-SSC — COMMENTED** [review]
Will consult with @stephanemillaire-ssc on usage of main and the impacts during pkg updates.

**Review by alaincormier-ssc — DISMISSED** [review]
looks good!

File: docs/landing-zone-v2/onboarding-client.md (hunk 1 above)
Comment (davelanglois-ssc) [reply docs/landing-zone-v2/onboarding-client.md:44]: @Michael-McAuley-SSC I'll wait for a come back from you two. not resolving the conversation for now

Comment (Michael-McAuley-SSC) [reply docs/landing-zone-v2/onboarding-client.md:44]: Please advise using versions over main - obtained through the .release-please-manifest.json. Alternatively, each package changelog contains the history if there is a requirement to use an older version.

Comment (cartyc) [reply docs/landing-zone-v2/onboarding-client.md:44]: Agreed with this. Main would be our equivalent of building from source and while that's possible it shouldn't be the first option and version should be preferred.

Comment (davelanglois-ssc) [reply docs/landing-zone-v2/onboarding-client.md:45]: latest commit include the new recommendation

File: docs/landing-zone-v2/README.md [hunk 2]
```diff
@@ -339,7 +340,8 @@ You will be running the following commands from a linux machine. GCP Cloud Shell
 
 We will be using `kpt` to obtain the packages. For more information on the `kpt get` command, please refer to this link : <https://kpt.dev/reference/cli/pkg/get/>
 
-> **!!! Update the command below with the proper VERSION, you can locate it in the package's CHANGELOG.md, for example, '0.0.1'. Use 'main' if not available**
+> **!!! Update the command below with the proper VERSION, you can locate it in the package's CHANGELOG.md, for example, '0.0.1'. Use 'main' if not available but
```
Comment (fmichaelobrien) [inline docs/landing-zone-v2/README.md:343]: reviewed updates

**Review by fmichaelobrien — APPROVED** [review]
LGTM

**Review by cartyc — APPROVED** [review]

**Pull request merged**